CHANGELOG.md

```diff
@@ -10,6 +10,25 @@
 - Fix issue where requesting a different format (e.g. `text/turtle`) of a
   JSON-LD resource crashed the server
 
+#### 3.5.0 Upgrade Notes
+
+- New config parameter: `serverUri` - Solid server uri (with protocol,
+  hostname and port), defaults to `https://localhost:8443`. In multi-user
+  (`"idp": true`) mode, new account directories are now created based on this
```

Comment:
I'd certainly prefer `multiuser`, myself. Maybe we can make the change backwards-compatible (so that existing configs also work with the `idp` setting)